api/capi/include/nnstreamer-single.h

```diff
@@ -96,9 +96,9 @@ int ml_single_open (ml_single_h *single, const char *model, const ml_tensors_inf
  * @param[in] hw Tell the corresponding @a nnfw to use a specific hardware.
  *               Set #ML_NNFW_HW_ANY if it does not matter.
  * @param[in] custom_option Comma separated list of options.
- *                      It is necessary to optimize the control for some neural network framework. (e.g. NumThreads:N to set the number of threads in TensorFlow-Lite)
+ *                      Use this parameter to fine-tune and optimize specific neural network framework. (e.g. NumThreads:N to set the number of threads in TensorFlow-Lite)
  *                      You may set NULL if it's not required.
- *                      See NNStreamer (https://github.com/nnstreamer/nnstreamer) documentation for the details.
+ *                      See NNStreamer documentation (https://nnstreamer.github.io/gst/nnstreamer/tensor_filter/README.html) for the details.
```

Comment:
How about linking this instead of tensor_filter's README.
- https://nnstreamer.github.io/Documentation/nnstreamer_capi.html#single-api